File: Array/easy/2011. Final Value of Variable After Performing Operations/finalValueAfterOperations.cpp
//time complexity : O(n)
//space complexity : O(1)
class Solution 
{
public:
    int finalValueAfterOperations(vector<string>& operations) 
    {
        int x = 0;
        for (string i : operations)
        {
            if (i == "X++" || i == "++X") x++;
            else x--;
        }
        return x;
    }
};

File: Array/easy/2011. Final Value of Variable After Performing Operations/finalValueAfterOperations.py
#time complexity : O(n)
#space complexity : O(1)
class Solution(object):
    def finalValueAfterOperations(self, operations):
        x = 0
        for i in operations:
            if i == 'X++' or i == '++X':
                x += 1
            elif i == '--X' or i == 'X--':
                x -= 1
                
        return x